webpack.server.config.js

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable comma-dangle, no-var, quote-props, vars-on-top
+*/
+
+var fs = require('fs')
+var path = require('path')
+
+var webpack = require('webpack')
+
+var nodeModules = {}
+fs.readdirSync('node_modules')
+  .filter(x => ['.bin'].indexOf(x) === -1)
+  .forEach(mod => { nodeModules[mod] = `commonjs ${mod}` })
+
+var config = {
+  cache: false,
+  entry: './server.js',
+  output: {
+    path: path.join(__dirname, 'build'),
+    filename: 'server.js'
+  },
+  target: 'node',
```

Comment:
I found that when I didn't have `target` set, things like `__dirname` wouldn't work properly, which was needed for webpack. I'll take another look as I go through and address your feedback from above.